controllers/articles.controller.js

```diff
@@ -40,4 +41,14 @@ exports.postCommentForArticle = (request, response, next) => {
      response.status(201).send({comment: comment});
    })
    .catch(next);
-};
\ No newline at end of file
+};
+
+exports.patchArticleByArticleId = (request, response, next) => {
+  const articleId = request.params.article_id;
+  const { inc_votes } = request.body;
```

Comment:
Keep some consistency in your destructuring. The article_id would be ```const {article_id} = request.params``` I don't mind if you use dot notation, or destructuring, but don't use both.